tensorflow-core/tensorflow-core-api/src/main/java/org/tensorflow/ConcreteFunction.java

```diff
@@ -109,15 +133,15 @@ public static ConcreteFunction create(Function<Ops, Signature> functionBuilder)
    * @return a new function
    */
   public static ConcreteFunction create(Signature signature, Graph graph) {
-    return new ConcreteFunction(signature, graph, new Session(graph), Ownership.SESSION_ONLY);
+    return buildFromGraph(graph, signature);
```

Comment:
It's a bit hard to understand it at a first glance but the fact that we don't keep track anymore of the resource ownership (`Ownership.*`) and that the unit tests that were validating it have been deleted in this PR raises some doubts.

Are we sure no resources will leak or closed prematurely if the function is being loaded from a saved model?

Reply:
See the new saved model function.  Creating a function from a graph creates the `TF_Function` object which has a separate lifetime, `ConcreteFunction`s don't have references to anything else.